packages/api/config/lambdas.yml

```diff
@@ -140,7 +140,10 @@ cleanExecutions:
   source: 'node_modules/@cumulus/api/dist/'
   namedLambdaDeadLetterQueue: true
   envs:
-    executionPayloadRetentionPeriod: '{{execution_payload_retention_period}}'
+    nonCompleteExecutionPayloadTimeout: '{{non_complete_execution_payload_timeout}}'
```

Comment:
I'm mildly unhappy with either label for these configuration flags, for both typo and/or clarity reasons, but nothing better came to mind.    Please feel free to suggest something more intuitive.

Reply:
Breaking the name into parts, I think `nonComplete` and `ExecutionPayload` are good descriptors, but I'm not sure about `Timeout`. I like `RetentionPeriod`, at the risk of being too long...

`nonCompleteExecutionPayloadRetentionPeriod`, while quite long, is more accurate from my perspective.

Reply:
I was a bit less concerned about the internal reference and more so about the user facing config name.  

🤔  

The problem with nonCompleteExecutionPayloadRetentionPeriod is it tends to run fowl of our style guidelines (line length) and as such winds up being immediately re-assigned to something shorter, which may or may not be desirable.   

It also is quite a mouthful, and while descriptive feels like it might not improve readability.    Then again, the other presented options have that same issue.